eng/ci/templates/jobs/build-artifacts.yml

```diff
@@ -0,0 +1,55 @@
+parameters:
+  - name: SkipBuildArtifacts
+    type: boolean
+    default: false
+
+jobs:
+
+- job: BuildArtifacts
+  displayName: Build Artifacts
+
+  pool:
+    name: 1es-pool-azfunc
+    image: 1es-windows-2022
+    os: windows
+
+  variables:
+    - template: /eng/ci/templates/variables/build.yml@self
+
+  templateContext:
+    outputParentDirectory: $(System.DefaultWorkingDirectory)/packages
+    outputs:
+    - ${{ if eq(parameters.SkipBuildArtifacts, false) }}:
+      - output: pipelineArtifact
+        displayName: Publish artifacts
+        path: $(System.DefaultWorkingDirectory)/packages
+        artifact: drop
+
+  steps:
+  - pwsh: dotnet --version
+    displayName: "Echo dotnet version"
+  
+  - task: DotNetCoreCLI@2
+    displayName: 'Build project'
+    inputs:
+      command: 'build'
+      arguments: '-c $(buildConfiguration)'
+      projects: 'Azure.Web.DataProtection.sln'
+
+  - task: DotNetCoreCLI@2
+    displayName: 'Run tests'
+    inputs:
+      command: 'test'
+      arguments: '--no-build -c $(buildConfiguration)'
+      projects: |
+        test/**/**Tests.csproj
```

Comment:
If `Azure.Web.DataProtection.sln` includes the test projects, you can use that here. `dotnet test` will be able to use that solution to find test projects and ignore non test projects.

Reply:
Done